### PR TITLE
Use omero.util.Resources to keep imports alive

### PR DESCRIPTION
**Pull request by joshmoore**
Recently reported by Christian Sachs, when `--auto_close` is
used, long-running imports may die in unusual ways. This maintains
a keep-alive as long as the `ManagedImportRequestI` servant is
open.

Testing this was facilitated by placing an artificial sleep in

```
@@ -1593,6 +1593,12 @@ public class OMEROMetadataStoreClient
     public void setPixelsFile(long pixelsId, String file, String repo) throws ServerError {
```

and setting the default session timeout to 10 seconds.

See:
http://lists.openmicroscopy.org.uk/pipermail/ome-users/2016-January/005791.html

### CODE REVIEW

File: components/blitz/src/ome/services/blitz/repo/ManagedImportRequestI.java [hunk 1]
```diff
@@ -261,6 +275,41 @@ public void init(Helper helper) {
         }
     }
 
+    /**
+     * If the {@link #setResources()} has been called with a non-null
```
Comment (sbesson) [inline components/blitz/src/ome/services/blitz/repo/ManagedImportRequestI.java]: From https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-merge-build/233/warnings39Result/

```
Tag @link: can't find setResources() in ome.services.blitz.repo.ManagedImportRequestI
```

`#setResources(Resources)` instead should fix this

Comment (joshmoore) [inline components/blitz/src/ome/services/blitz/repo/ManagedImportRequestI.java]: Pushed.

**Comment by mtbc** [comment]
Will be tested by today's importing.

File: components/blitz/src/ome/services/blitz/repo/ManagedImportRequestI.java [hunk 2]
```diff
@@ -261,6 +275,41 @@ public void init(Helper helper) {
         }
     }
 
+    /**
+     * If the {@link #setResources(Resources)} has been called with a
+     * non-null value, then create an {@link Entry} which will ping the
+     * {@link ServiceFactoryPrx} instance periodically to keep the import
+     * from timing out. This is especially important for autoClose imports
+     * since no other client is likely to keep them alive.
```
Comment (mtbc) [inline components/blitz/src/ome/services/blitz/repo/ManagedImportRequestI.java]: :+1: for explaining motivation

**Comment by mtbc** [comment]
This code looks good to me. Would be nice to get https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-merge-integration-python/lastCompletedBuild/testReport/ green though even if it's not this PR's fault.

**Comment by joshmoore** [comment]
@mtbc : agreed -- no rush to merge until the builds are happy.

**Comment by joshmoore** [comment]
After the re-run, jobs are green (barring one unrelated timeout):

 * https://ci.openmicroscopy.org/job/OMERO-DEV-merge-integration-python/206/
 * https://ci.openmicroscopy.org/job/OMERO-DEV-merge-integration-java/205/

**Comment by joshmoore** [comment]
Pushed.

**Comment by mtbc** [comment]
Good to merge.

**Pull request merged**